tutorials/identifying-first-day-no-snow/index.md

```diff
@@ -0,0 +1,516 @@
+_Patterns of snowmelt timing are represented here by the first day in 2018 where
+each pixel no longer contained snow, as detected by the MODIS daily snow cover
+product. The color grades from purple (earlier) to yellow (later) as the first
+day of the year with zero percent snow per pixel increases
+(Arctic Polar Stereographic projection)._
+
+## Context
+
+The timing of annual seasonal snowmelt and any potential change in that
+timing has broad ecological implications and thus impacts human livelihoods,
+particularly in and around high latitude and mountainous systems.
+
+One of the most important phases of the hydrologic cycle within these
+regions, the
+annual melting of accumulated winter snowfall provides the dominant source of
+water for streamflow and groundwater recharge for approximately one sixth of
+the global population (Musselman et al., 2017; Barnhart et al., 2016;
+Bengtsson, 1976).
+
+The timing of snowmelt in the Arctic and Antarctic influences the length of
+the growing season; and consistent snow cover
+throughout the winter insulates vegetation from harsh temperatures and wind
+(Duchesne et al., 2012; Kudo et al., 1999). In mountainous regions such as
+the Himalayas, snowmelt is a major source of freshwater downstream (Barnhart
+et al., 2016) and is essential in recharging groundwater.
+
+This seasonal water resource is one of the fastest changing hydrologic
+systems under Earth’s
+warming climate, and will broadly impact regional economies, ecosystem
+function and increase the potential for flood hazards (Musselman et al.,
+2017; IPCC, 2007; Beniston, 2012; Allan and Casillo, 2007; Barnett and
+Lettenmaier, 2005).
+
+The anticipated warmer temperatures will alter the type
+and onset of precipitation; multiple regions, including the Rocky Mountains
+of North America have already measured a reduction in snowpack volume and
+warmer temperatures have shifted precipitation from snowfall to rain, causing
+snowmelt to occur earlier (Barnhart et al., 2016; Clow, 2010; Harpold et al.,
+2012).
+
+This tutorial calculates the first day of no snow annually at the pixel
+level, providing the user with the ability track the seasonal and interannual
+variability in the timing of snowmelt toward a better understanding of how
+the hydrological cycles of higher latitude and mountainous regions are
+responding to climate change.
+
+## Identify first day of zero percent snow cover
+
+This section covers building an image collection where each image is a mosaic
+of pixels that describe the first day in a given year that zero percent snow
+cover is recorded. Snow cover is defined by the [MODIS NDSI Snow Cover
+product](https://developers.google.com/earth-engine/datasets/catalog/MODIS_006_MOD10A1).
+The general workflow is as follows:
+
+1. Define the date ranges to consider for analysis.
+2. Define a function that adds date information as bands to snow cover images.
+3. Define an analysis mask.
+4. Loop through years; for each year:
```

Comment:
Nit/opinion: Some may misinterpret "loop through" to mean using a "for" loop client side, while what we're really doing here is mapping a function over a collection. In that case, it might be safer to say simply "For each year:".

Reply:
Done